main.go

```diff
@@ -118,11 +118,11 @@ func main() {
 	// parse flags
 	pflag.StringVar(&configPath, "config", "", "Configuration file path")
 
-	pflag.StringVar(&cfg.Controller.MetricsAddr, "metrics-addr", ":8080", "The address the metric endpoint binds to.")
-	handleFatalError(viper.BindPFlag("controller.metrics.addr", pflag.Lookup("metrics-addr")))
+	pflag.StringVar(&cfg.Controller.MetricsAddr, "metrics-bind-address", ":8080", "The address the metric endpoint binds to.")
```

Comment:
Just a nitpick, `.MetricsAddr` and `metrics-bind-address`, Maybe we should change to `.MetricsBindAddr`

Reply:
Great suggestion